keyboards/ergodox_ez/keymaps/bdk/keymap.c

```diff
@@ -0,0 +1,176 @@
+#include QMK_KEYBOARD_H
+
+#define SHIFT_DOWN() (get_mods() & (MOD_BIT(KC_LSFT)|MOD_BIT(KC_RSFT)))
+#define KC_FKEYS MO(FKEYS)
+
+// Layers
+enum {
+        BASE=0,
```

Comment:
This is oddly indented.

```suggestion
  BASE=0,
```